docs/OnnxReleases.md

```diff
@@ -13,7 +13,7 @@ The ONNX project, going forward, will plan to release roughly on a four month ca
 * Before creating the release branch, increase `VERSION_NUMBER` in the main branch. The following files will be updated: [VERSION_NUMBER file](/VERSION_NUMBER) and
 [version.h](/onnx/common/version.h)
 
-* Please use a VERSION_NUMBER smaller than the target (release VERSION_NUMBER) and larger than the previous one to test TestPyPI before using the target VERSION_NUMBER. 
+* Please use VERSION_NUMBER with `rc` (e.g., `1.x.0rc1`) to test TestPyPI in advance before using the target VERSION_NUMBER for final release.
```

Comment:
Please use target VERSION_NUMBER with `rc` (e.g., `1.x.0rc1`) to test TestPyPI in advance before using it for final release.